packages/vm/src/testing/contract.rs

```diff
@@ -24,8 +24,9 @@ pub struct Contract {
 }
 
 /// representing a contract in integration test
-///
-/// This enables tests instantiate a new instance every time testing call_(instantiate/execute/query/migrate) like actual wasmd's behavior.
+/// This enables tests to instantiate a new instance every time,
```

Comment:
do not delete a blank line between a summary and a description in the doc.

Reply:
Fixed, thank you.